pkg/webhooks/server.go

```diff
@@ -342,17 +342,7 @@ func (ws *WebhookServer) resourceMutation(request *v1beta1.AdmissionRequest) *v1
 	// Success -> Generate Request CR created successsfully
 	// Failed -> Failed to create Generate Request CR
 	if request.Operation == v1beta1.Create || request.Operation == v1beta1.Update {
-		ok, msg := ws.HandleGenerate(request, generatePolicies, ctx, userRequestInfo)
-		if !ok {
-			logger.Info("admission request denied")
-			return &v1beta1.AdmissionResponse{
-				Allowed: false,
-				Result: &metav1.Status{
-					Status:  "Failure",
-					Message: msg,
-				},
-			}
-		}
+		go ws.HandleGenerate(request, generatePolicies, ctx, userRequestInfo)
```

Comment:
do we need to operate on a copy of the request? Is it thread-safe to pass in the original request?

Reply:
Same with userRequestInfo - if these are immutable we are ok,

Reply:
Change to pass in the copy of the request. There is only Read operation on those parameters.